public/app/plugins/datasource/azuremonitor/components/LogsQueryBuilder/AzureMonitorKustoQueryParser.ts

```diff
@@ -0,0 +1,120 @@
+import { BuilderQueryEditorWhereExpressionArray, BuilderQueryExpression } from '../../dataquery.gen';
```

Comment:
This file should be called query builder instead of parser right>

Reply:
That works :)